source/MBF.SAMD51.SystemCore.pas

```diff
@@ -25,6 +25,35 @@ interface
 {$INCLUDE MBF.ARM.SystemCore.inc}
 {$UNDEF INTERFACE}
 
+{$ifdef freertos_fat}
+
+  {$LINKLIB libfreertos_wio.a}
+  {$LINKLIB libstdc++_nano.a,static}
+  {$LINKLIB libm.a,static}
+  {$LINKLIB libc_nano.a,static}
+  {$LINKLIB libgcc.a,static}
+  {$LINKLIB libnosys.a,static}
+
+  procedure BasicSystemInit; cdecl; external name 'SAMD51SystemInit';
```

Comment:
For what rerason do you need those 3 externalls?

Reply:
See: 

https://github.com/LongDirtyAnimAlf/mbf-freertos-wio/blob/master/SamplesBoardSpecific/WioTerminal/Hardware/extras/System/samd51_system_init.c#L51

https://github.com/LongDirtyAnimAlf/mbf-freertos-wio/blob/master/SamplesBoardSpecific/WioTerminal/Hardware/extras/System/samd51_system_init.c

https://github.com/LongDirtyAnimAlf/mbf-freertos-wio/tree/master/SamplesBoardSpecific/WioTerminal/Hardware/extras

Reply:
Its the external init of the Arduino hardware library.